builders/integrations/analytics/dapplooker.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Dapplooker
+description: 使用Dapplooker分析和查询链上数据，并创建数据面板为Moonbeam和Moonriver可视化数据分析。
+---
+
+# 开始使用Dapplooker
+
+![Dapplooker Banner](/images/builders/integrations/analytics/dapplooker/dapplooker-banner.png)
+
+## 概览 {: #introduction }
+
+Moonriver和Moonbeam上的开发者可以使用[Dapplooker](https://dapplooker.com/){target=_blank}平台分析其链上数据并运行SQL查询。此集成使项目能够创建图表和数据面板以追踪其智能合约并提供性能和采用方面的反馈。
+
+Dapplooker分析平台协助基于Moonbeam的网络，帮助用户无需依赖工程师或分析师即可理解智能合约。Dapplooker直观的Visual SQL有助于以表格形式浏览智能合约数据，并使用易于使用的编辑器编写SQL查询。用户可以创建、分叉和与所有人共享数据面板。
+
+本教程将涵盖通过Dapplooker平台注册项目所需的所有详情以及如何使用SQL编辑器构建分析。本教程适用于Moonbeam、Moonriver和Moonbase Alpha网络。
+
+## 查看先决条件 {: #checking-prerequisites }
+
+在开始操作之前，您将需要创建一个Dapplooker账户。您可以[点击此处注册](https://dapplooker.com/signup){target=blank}并创建一个账户。如果您已注册，您可以直接[点击此处登录](https://dapplooker.com/login){target=blank}您的账户。
+
+![Login to Dapplooker](/images/builders/integrations/analytics/dapplooker/dapplooker-1.png)
+
+## 连接智能合约 {: #connect-smart-contracts }
+
+要将智能合约连接至Dapplooker，您可以点击页面上方的**My Projects**按钮。
+
+在**Register Your Project**页面，点击**Connect Dapp**，然后选择**Connect Smart Contract**选项。您也可以从菜单栏中浏览和运行已编入索引的DApp分析。
+
+![Connect dapp](/images/builders/integrations/analytics/dapplooker/dapplooker-2.png)
+
+随后，系统将提示您输入您项目和合约的详情：
+
+1. 输入您的项目名称
+
+2. 输入您项目运行所在的网络。网络可以选择**Moonbeam**、**Moonriver**或**Moonbase Alpha**
+
+3. 如果您有多个相同合约的实例，你可以启动按钮
+
+4. 输入您的合约地址。如果合约地址已在[Moonscan](https://moonscan.io/){target=_blank}验证，将在启动自动完成时出现。您可以在自动完成处选择合约地址。如果在自动完成时未出现合约地址，您可以输入您的合约地址，点击**+**按钮，然后点击Click To Upload按钮上传ABI
```

Comment:
@hyd628 can you please verify this is correct?

Reply:
changed this slightly.